test/Ocelot.UnitTests/DownstreamUrlCreator/DownstreamUrlCreatorMiddlewareTests.cs

```diff
@@ -223,7 +223,7 @@ public void should_create_service_fabric_url()
             this.Given(x => x.GivenTheDownStreamRouteIs(downstreamRoute))
                 .And(x => GivenTheServiceProviderConfigIs(config))
                 .And(x => x.GivenTheDownstreamRequestUriIs("http://localhost:19081"))
-                .And(x => x.GivenTheUrlReplacerWillReturn("/api/products/1"))
+                .And(x => x.GivenTheUrlReplacerWillReturn("Ocelot/OcelotApp/api/products/1"))
```

Comment:
Is there a reason to add ```Ocelot/OcelotApp``` to all these tests

Reply:
Thanks for your response.
The reason is, that `_replacer.Replace()` now gets called twice. Once for the `context.DownstreamReRoute.DownstreamPathTemplate` and once for the ServiceFabric service name.

I've just updated the commit to make it more clear. Now it looks like follows:

```csharp
.And(x => x.GivenTheUrlReplacerWillReturnSequence("/api/products/1", "Ocelot/OcelotApp"))
```
The first parameter `/api/products/1` gets returned when replacing the placeholders in `context.DownstreamReRoute.DownstreamPathTemplate`. 
The second parameter `Ocelot/OcelotApp` when replacing the placeholders in the ServiceFabric service name